src/helpers/theme_provider.tsx

```diff
@@ -61,8 +47,10 @@ export const ThemeProvider = (props: ThemeProviderProps) => {
 export type ThemeDecoratorContext = { globals: { theme: Theme } }
```

Comment:
I guess we don't need this anymore then..?

Reply:
Good find!

Reply:
done